src/main/java/pro/sky/telegrambot/listener/TelegramBotUpdatesListener.java

```diff
@@ -3,22 +3,41 @@
 import com.pengrad.telegrambot.TelegramBot;
 import com.pengrad.telegrambot.UpdatesListener;
 import com.pengrad.telegrambot.model.Update;
+import com.pengrad.telegrambot.request.SendMessage;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
 import org.springframework.stereotype.Service;
+import pro.sky.telegrambot.model.notificationTtask;
+import pro.sky.telegrambot.repository.RepositoryNotification_task;
+import pro.sky.telegrambot.sevices.NotificationTtaskService;
 
 import javax.annotation.PostConstruct;
+import java.text.ParseException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+import java.util.Collection;
 import java.util.List;
 
 @Service
 public class TelegramBotUpdatesListener implements UpdatesListener {
 
     private Logger logger = LoggerFactory.getLogger(TelegramBotUpdatesListener.class);
+    @Autowired
+    private RepositoryNotification_task repositoryNotification_task;
 
     @Autowired
     private TelegramBot telegramBot;
 
+    @Autowired
+    private NotificationTtaskService notificationTtaskService;
```

Comment:
опечатка в имени класса
```suggestion
    private NotificationTaskService notificationTaskService;
```